src/main/java/uk/gov/register/util/DateFormat.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.register.util;
+
+public interface DateFormat {
+
+    interface ISO_8601 {
+        String LABEL = "ISO_8601";
+
+        String YEAR = "yyyy";
+        String YEAR_MONTH = "yyyy-MM";
+        String YEAR_MONTH_DAY = "yyyy-MM-dd";
+        String YEARS_MONTH_DATA_TIME = "yyyy-MM-dd'T'hh:mm:ss";
```

Comment:
This name and the one after is a bit confusing with the `YEARS` plural. How about instead `YEAR_MONTH_DAY_HOURS_MINUTES_SECONDS`?